src/api/controllers/messages.ts

```diff
@@ -85,6 +85,8 @@ export async function handleInternalMessageUpdate(data: DiscordPayload) {
   const cachedMessage = await cacheHandlers.get("messages", payload.id);
   if (!cachedMessage) return;
 
+  cachedMessage.content = payload.content;
```

Comment:
This should bemuch lower likeone of the last things that is done in this function. If you overwrite it this early everything below will break